src/content/recipes/@emotion/styled.md

```diff
@@ -39,13 +19,14 @@ Run the following script to install and register the addon:
 npx storybook@latest add @storybook/addon-themes
 ```
 
-### How to setup `GlobalStyles`
+<details>
+  <summary>Did the configuration script fail?</summary>
+  <p>Under the hood, this runs <code>npx @storybook/auto-config themes</code>, which should read your project and try to configure your Storybook with the correct decorator. If running that command directly does not solve your problem, please file a bug on the <a href="https://github.com/storybookjs/auto-config/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=%5BBug%5D" target="_blank">@storybook/auto-config</a> repository so that we can make this as good as possible. To manually add this addon, install it, and then add it to the addons array in your <code>.storybook/main.ts</code>.</p>
+</details>
 
-UIs often have a set of global styles that are applied to every component like CSS resets, `font-size`, `font-family`, and colors.
+## 2. Provide `GlobalStyles`
 
-In Emotion, use the [`Global`](https://emotion.sh/docs/globals) component to scope styles globally instead of locally (which is the library's default behavior).
-
-Open `.storybook/preview.js` and create a `GlobalStyles` component which includes a `font-family`. Then apply it to your stories with the [`withThemeFromJSXProvider`](https://github.com/storybookjs/storybook/blob/next/code/addons/themes/docs/api.md#withthemefromjsxprovider) decorator by adding it to the `decorators` array.
+Inside of `.storybook/preview.js`, create a `<GlobalStyles />` component that includes a `font-family`. Then apply it to your stories with the [`withThemeFromJSXProvider` decorator](https://github.com/storybookjs/storybook/blob/next/code/addons/themes/docs/api.md#withthemefromjsxprovider) by adding it to the decorators array.
 
 ```js
 // .storybook/preview.js
```

Comment:
This will likely need to use the `jsx` file extension, because of the use of JSX.

Is that (`jsx` being different than the auto-generated `js`, likewise for `tsx` and `ts`) worth a Callout? I think it probably is.

```suggestion
// .storybook/preview.jsx
```